docker-entrypoint.sh

```diff
@@ -1,11 +1,17 @@
 #!/bin/sh
 # Note: This file MUST have LF line endings
 
+# While Docker doesn't actually use the postgres-info.json file, it must be filled out
+# otherwise, autoscheduler/config/config.py will call sys.exit() and end the tests before they can run
+echo "Creating empty postgres-info.json file"
+echo "{ \"user\": \"a\", \"password\": \"a\" }" > /app/autoscheduler/config/postgres-info.json
+
 echo "Creating migrations"
 python3 manage.py makemigrations --settings=autoscheduler.settings.docker scraper
 
 echo "Migrating"
 python3 manage.py migrate --settings=autoscheduler.settings.docker scraper
 
 echo "Starting tests"
-python3 manage.py test --settings=autoscheduler.settings.docker
+export DJANGO_SETTINGS_MODULE=autoscheduler.settings.docker
```

Comment:
This export overrides the `DJANGO_SETTINGS_MODULE=autoscheduler.settings.base` line in `pytest.ini`, so we don't have to make another `pytest.ini` only for Docker to use.